Makefile

```diff
@@ -106,7 +108,14 @@ container-build: unittest bundle
 	mkdir -p data/crd
 	cp bundle/manifests/ssp-operator.clusterserviceversion.yaml data/olm-catalog/ssp-operator.clusterserviceversion.yaml
 	cp bundle/manifests/ssp.kubevirt.io_ssps.yaml data/crd/ssp.kubevirt.io_ssps.yaml
-	docker build -t ${IMG} .
+	docker build -t ${IMG} \
+		--build-arg IMG_REPOSITORY=${IMG_REPOSITORY} \
```

Comment:
Do these build arguments need to be defined inside the Dockerfile using `ARG` instruction?

Reply:
Yes, you are right, I thought it was implicit, but it isn't, fixed now